lib/src/css_parser.dart

```diff
@@ -418,4 +539,29 @@ class ExpressionMapping {
       return null;
     }
   }
+
+  static Color hslToRgbToColor(String text) {
+    final hslText = text.replaceAll(')', '').replaceAll(' ', '');
```

Comment:
What's happening here? At the very least this needs some comment with an example. But really this should have some unit tests.

Reply:
HSL Text input: `hsl(39, 100%, 50%)`
CSS Parser package output: `39, 100%, 50%)`

The first line removes the extra spaces and the trailing ) so the values can be extrapolated into a list. We do the exact same thing for the `rgbaToColor` function.